### PR TITLE
Add Lodata.md

### PR DESCRIPTION
**Pull request by 27pchrisl**
Lodata is a fully-featured OData 4.01 producer for the Laravel web application framework.

### CODE REVIEW

**Comment by 27pchrisl** [comment]
Hi @marabooy - Is there anything else to be provided for this submission, or is it just waiting until people have free time to review?

**Comment by marabooy** [comment]
@27pchrisl thanks for the notification, I should be able to publish this soon once reviewed.

**Review by marabooy — APPROVED** [review]

**Review by ElizabethOkerio — APPROVED** [review]

File: _libraries/Lodata.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+link: https://github.com/flat3/lodata
+version: V4 
+object: Server
+downloads:
+  - source: GitHub
+    link: https://github.com/flat3/lodata
+  - source: Packagist
+    link: https://packagist.org/packages/flat3/lodata
+---
+Lodata is a fully-featured OData 4.01 producer for the Laravel web application framework.
```
Comment (KenitoInc) [inline _libraries/Lodata.md:13]:
```suggestion
Lodata is a fully-featured OData v4.01 producer for the Laravel web application framework.
```

**Review by KenitoInc — APPROVED** [review]
A minor suggestion.

File: _libraries/Lodata.md (hunk 1 above)
Comment (marabooy) [reply _libraries/Lodata.md:13]: @27pchrisl could you add this suggestion and so I can publish Asap.

**Comment by 27pchrisl** [comment]
@marabooy does the website get regenerated automatically ?

**Comment by marabooy** [comment]
@27pchrisl the changes are now live.

**Pull request merged**